app/controllers/trips_controller.rb

```diff
@@ -1,4 +1,5 @@
 class TripsController < ApplicationController
+  before_action :set_trip, only: [:show, :edit, :update, :destroy]
```

Comment:
is there a performance gain to doing it this way vs how it was?

Reply:
just following the DRY principle because it was repeating 4 times with same line of code